.github/workflows/ghcr.yml

```diff
@@ -66,8 +68,10 @@ jobs:
 
       - name: Build and push ${{ matrix.image }}
         if: "!github.event.pull_request.head.repo.fork"
```

Comment:
If you are splitting this action into two, could you also do me a favor and fix this? We currently do push for every branch on opendevin repo. We shouldn't! We should only push the image on opendevin::main.